db-client-java/src/main/java/com/eventstore/dbclient/PersistentSubscriptionStats.java

```diff
@@ -0,0 +1,121 @@
+package com.eventstore.dbclient;
+
+public class PersistentSubscriptionStats {
+    private double averagePerSecond;
+    private long totalItems;
+    private long countSinceLastMeasurement;
+    private long lastCheckpointedEventRevision;
+    private long lastKnownEventRevision;
+    private Position lastCheckpointedPosition;
```

Comment:
**FIX:** Based on the RFC it should be `lastCheckpointedEventPosition`
```suggestion
    private Position lastCheckpointedEventPosition;
```

Reply:
I think that this should be reflected in RFC.

Reply:
This has been fixed by splitting the stats into two types, one for $all and one for streams

Reply:
![one for $all](https://c.tenor.com/_NF22paU3x0AAAAC/all-for-one-all-for-one-and-one-for-all.gif)